### DISCUSSION TITLE
How to disable method/function signature popover in editor?

### DISCUSSION THREAD

**Discussion by gitdode** [apache/netbeans#6730] [Q&A] [answered]
When scrolling in a source file in the editor, I sometimes get a popup in the top left area, showing the signature of the method/function that is currently highlighted:

![grafik](https://github.com/apache/netbeans/assets/11530253/d5808b39-c4bb-49f0-9575-222ab7f563a8)

While this is probably a useful feature for many, it kinda distracts me and I tend to click it away all the time.
How can I turn this off? I looked through the settings but I wouldn't even know how this feature is called and where to find it.

Cheers,
Torsten

**Comment by JoachimRohde** [comment, marked as answer]
This happens if the cursur is placed next to a closing brace and the opening brace is not in the viewport. You can disable this in the options (editor/general/show tooltips):
![netbeans-show-tooltips](https://github.com/apache/netbeans/assets/1743626/8ee1f31d-dd63-47ad-a48b-04833991a611)

**Reply by gitdode** [reply]
That was it, thank you!
